src/devices/Mcp23xxx/tests/Mcp23xxxTest.cs

```diff
@@ -120,8 +123,9 @@ public Mcp23xxxChipMock(int ports, bool isSpi)
             public Span<byte> Registers => _registers;
 
             // Can't coalesce here https://github.com/dotnet/roslyn/issues/29927
-            public ReadOnlySpan<byte> LastReadBuffer => _lastReadBuffer == null ? ReadOnlySpan<byte>.Empty : _lastReadBuffer;
-            public ReadOnlySpan<byte> LastWriteBuffer => _lastWriteBuffer == null ? ReadOnlySpan<byte>.Empty : _lastWriteBuffer;
+            public ReadOnlySpan<byte> LastReadBuffer => _lastReadBuffer ?? ReadOnlySpan<byte>.Empty;
```

Comment:
This is the issue that is causing the build break. As you see in the comment above, there is an existing compiler issue causing the 'Failed to emit module' error when you try to Coalesce with the ?? operator. I tried building with all your changes + reverting these two lines and building works fine for me now.

Reply:
Ah, good catch.  And I remember seeing that when changing.  Little confused on why it worked on my end, though.  Anyways, builds look like they're working now.  Thx for finding this.